test/resources/minimal-sample-1_6_x/components-definition.json

```diff
@@ -0,0 +1,97 @@
+{
+    "name": "minimal-sample-1-6-x",
+    "description": "Minimal components package sample touching most of the available options.",
+    "version": "1.6.0",
+    "defaultComponentOnEnter": "body",
+
+    "components": [
+        {
+            "name": "body",
+            "label": "Body Label",
+            "icon": "icons/component.svg",
+            "properties": ["selectProperty"],
+            "countStatistics": true
+        },
+        {
+            "name": "intro",
+            "label": {
+                "key": "INTRO_KEY"
+            },
+            "icon": "icons/component.svg",
+            "properties": ["checkboxProperty"],
+            "directiveOptions": {
+                "text": {
+                    "stripStylingOnPaste": false
```

Comment:
I added the option here but noticed that there are not a lot of options set in the minimal sample. So is it ok to add it here or should we just rely on the individual validator tests?

Reply:
Should be ok I think?